align.go

```diff
@@ -205,51 +245,77 @@ func (a *Align) export(lines []string) {
 				}
 			}
 
-			word = applyPadding(word, tempColumn, a.columnCounts[columnNum], j, string(p))
+			padLength := countPadding(word, a.columnCounts[columnNum])
+			// a.padder.Grow(padLength + len(word) + (len(surroundingPad) * 2)) // TODO: might be able to do this once
+
+			paddedWord := applyPadding(a.padder, word, string(surroundingPad), tempColumn, padLength, j)
```

Comment:
What's the overhead of this conversion to string? Is it relevant?

Reply:
Are you suggesting we just return the `[]byte` instead?  I was sort of going that direction, and this refactor would have consisted of using a `strings.Builder` to avoid allocating a string on return.  However, I soon realized that I couldn't truncate (ie. my goal is to keep the underlying array around).

Reply:
In anticipation, it is done.  Allocations reduced even further.